src/components/Organizations/Organizations.js

```diff
@@ -11,11 +12,13 @@ const Organizations = props => {
 
   return (
     <Breadcrumb data={{ title: 'ORGANIZATIONS', pathname: url }}>
-      <Switch>
-        <Route component={List} exact path={`${path}`} />
-        <Route component={Detail} path={`${path}/:orgId`} />
-        <Redirect path={`${path}*`} to={`${url}`} />
-      </Switch>
+      <LoadingOverlay loading={props.loadingClustersDetails !== false}>
```

Comment:
Same suggestion. Unless I am missing something or having a boolean logic brainfart.

```suggestion
      <LoadingOverlay loading={props.loadingClustersDetails}>
```